packages/rfw/lib/src/flutter/argument_decoders.dart

```diff
@@ -1420,3 +1447,57 @@ class ArgumentDecoders {
     }
   }
 }
+
+// TODO(shuoch-g): Move this class to other places where more appropriate.
+/// An implementation of [SliverPersistentHeaderDelegate] that takes in fixed
+/// child widget.
+/// 
+/// Caller can configure [maxExtent] and [minExtent], but the returned widget
+/// in [build] is always [child] which is fixed (doesn't shrink).
+class FixedSliverPersistentHeaderDelegate
+    extends SliverPersistentHeaderDelegate {
+  /// Creates a [FixedSliverPersistentHeaderDelegate] delegate.
+  /// 
+  /// The [fixedMaxExtent], [fixedMinExtent] and [child] are all required,
+  const FixedSliverPersistentHeaderDelegate({
+    required this.fixedMaxExtent,
+    required this.fixedMinExtent,
+    required this.child,
+  });
+
+  /// Fixed `maxExtent`.
+  /// 
+  /// In [build], the [child] is fixed (not shrink according to this value).
+  final double fixedMaxExtent;
+
+  /// Fixed `maxExtent`.
+  /// 
+  /// In [build], the [child] is fixed (not shrink according to this value).
+  final double fixedMinExtent;
+
+  /// Fixed widget to return in [build].
+  ///
+  /// In [build], it is fixed (not shrink).
+  final Widget child;
+
+  @override
+  Widget build(BuildContext context, double shrinkOffset, bool overlapsContent) {
+    return child;
+  }
+
+  @override
+  double get maxExtent => fixedMaxExtent;
+
+  @override
+  double get minExtent => fixedMinExtent;
+
+  @override
+  bool shouldRebuild(covariant SliverPersistentHeaderDelegate oldDelegate) =>
+    fixedMaxExtent != oldDelegate.maxExtent || 
+      fixedMinExtent != oldDelegate.minExtent;
+
+  // TODO(shuoch-g): Add fixed configurations below.
+  // FloatingHeaderSnapConfiguration
+  // OverScrollHeaderStretchConfiguration
+  // PersistentHeaderShowOnScreenConfiguration
```

Comment:
we should either add these or just remove the comment here. There's an infinite number of things we don't yet support, no need to list them all. :-)

Reply:
removed them.

Yeah you are right. That's also why I would suggest we eventually keep this set of things in one place (RFW), it also benefits our project: it is much better if this set is in align with flutter team.